test/testcommon/src/main/java/io/pravega/test/common/AssertExtensions.java

```diff
@@ -45,8 +45,14 @@
     public static <T> void assertEventuallyEquals(T expected, Callable<T> eval, long timeout) throws Exception {
         long endTime = System.currentTimeMillis() + timeout;
         while (endTime > System.currentTimeMillis()) {
-            if (expected == eval.call()) {
-                return;
+            if (expected == null) {
+                if (eval.call() == null) {
+                    return;
+                }
+            } else {
```

Comment:
you can merge this into an `else if` block

Reply:
This comment hasn't been addressed, I have unresolved this conversation.

Reply:
it's fixed in commit 053715b493b